File: PythonCode/1046.py
#Questão: Tempo de jogo
a,b = map(int, input().split())
if a == b:
    print("O JOGO DUROU 24 HORA(S)")
elif b > a:
    x = abs(a - b)
    print("O JOGO DUROU",x,"HORA(S)")
elif b < a:
    x = (abs(a-24)+b)
    print("O JOGO DUROU",x,"HORA(S)")

File: C++Code/1046.cpp
//Questão: Tempo de jogo
#include <iostream>

using namespace std;

int main(){
    int hi,hf,tt;
    cin >> hi >> hf;

    if(hi == hf){
        printf("O JOGO DUROU 24 HORA(S)\n");
    }
    else if (hi > hf)
    {
        hf += 24;
        tt = hf -hi;
        printf("O JOGO DUROU %d HORA(S)\n", tt);
    }
    else
    {
        tt = hf - hi;
        printf("O JOGO DUROU %d HORA(S)\n", tt);
    }
    
    
}